bedrock/3_EntityWorkflow/workflows.py

```diff
@@ -0,0 +1,174 @@
+import asyncio
+from datetime import timedelta
+from temporalio import workflow
+from collections import deque
+from typing import List, Tuple, Deque, Optional
+from dataclasses import dataclass
+
+with workflow.unsafe.imports_passed_through():
+    from activities import prompt_bedrock
+
+
+@dataclass
+class BedrockParams:
+    conversation_summary: Optional[str] = None
+    prompt_queue: Optional[Deque[str]] = None
+
+
+@workflow.defn
+class EntityBedrockWorkflow:
+    def __init__(self) -> None:
+        # List to store prompt history
+        self.conversation_history: List[Tuple[str, str]] = (
+            []
+        )
+        self.prompt_queue: Deque[str] = deque()
+        self.conversation_summary: Optional[str] = None
+        self.continue_as_new_per_turns: int = 6
```

Comment:
We usually encourage using `workflow.info().is_continue_as_new_suggested()`, but don't have to

Reply:
Where did we land on this @steveandroulakis? For context, this is currently done this way so a/ developers can easily trigger CAN, b/ the token input limit to the Bedrock model may be lower than the Temporal history limit.

Reply:
I think manually setting the limit is preferred for those reasons. You might want to have a couple of # comments saying "set this lower to trigger CAN (for learning)" or something. And also even note the suggest..() function exists?

Reply:
I admit not digging into the sample, but token limit of some system should not affect CAN. But yes, if you are not using `is_continue_as_new_suggested()` because you want to CAN earlier just for the sample, ok, but yes it's worth mentioning. For most users it is not worth CAN'ing so frequently.